rocAL/CMakeLists.txt

```diff
@@ -297,10 +297,12 @@ if(${BUILD_ROCAL})
         target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_HIP=1)
         target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_OPENCL=0)
         if(ROCDECODE_FOUND)
-            target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_ROCDECODE=1)
-            message("-- ${White}rocAL built with rocDecode video decode capabilities${ColourReset}")
-        else()
-            target_compile_definitions(${PROJECT_NAME} PUBLIC ENABLE_ROCDECODE=0)
+            if(${ROCDECODE_VER_MAJOR} VERSION_GREATER_EQUAL "0" AND ${ROCDECODE_VER_MINOR} VERSION_GREATER_EQUAL "10" AND ${ROCDECODE_VER_MICRO} VERSION_GREATER_EQUAL "0")
+                target_compile_definitions(${PROJECT_NAME} PUBLIC -DENABLE_ROCDECODE)
```

Comment:
Shouldn't we set this to `ENABLE_ROCDECODE=1`?

Reply:
The -DENABLE_ROCDECODE is equivalent to ENABLE_ROCDECODE=1
Mentioned in the [doc](https://cmake.org/cmake/help/latest/command/target_compile_definitions.html):

> Note that many compilers treat -DFOO as equivalent to -DFOO=1